### PR TITLE
Remove .exe extension from pwsh in PostBuild

### PR DESCRIPTION
**Pull request by JunielKatarn**
Allows building in UNIX via .NET 6.

### CODE REVIEW

File: PoshMailKit/PoshMailKit.csproj [hunk 1]
```diff
@@ -59,7 +59,7 @@
   </ItemGroup>
   
   <Target Name="PostBuild" AfterTargets="PostBuildEvent">
-    <Exec Command="pwsh.exe -C &quot;&amp; '$(TargetDir)\Build\BuildScript.ps1'&quot;" />
+    <Exec Command="pwsh -C &quot;&amp; '$(TargetDir)\Build\BuildScript.ps1'&quot;" />
```
Comment (JunielKatarn) [inline PoshMailKit/PoshMailKit.csproj:62]: This will still work on Windows, but allow building in macOS, Linux.

Comment (poshcodebear) [reply PoshMailKit/PoshMailKit.csproj:62]: Okay, that's hilarious...not sure why those tabs were there on lines 10 and 11, but I'm going to blame Visual Studio for that one.  Good catch, thanks!

**Comment by poshcodebear** [comment]
Nice catch; I didn't consider that Linux doesn't have that extension, so thank you for this!

**Comment by JunielKatarn** [comment]
> Nice catch; I didn't consider that Linux doesn't have that extension, so thank you for this!

Thank YOU. I had been looking for a cross-platform command line email solution.
This looks like the most promising approach.

I'll try to contribute more as time permits.

**Pull request merged**